operator/tests/common/central-cr.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: stackrox-central-services
 spec:
   imagePullSecrets:
-  - name: e2e-test-pull-secret
+    - name: e2e-test-pull-secret
```

Comment:
🧐

Reply:
I don't know where this came from - it wasn't in my local client when I looked. I re-pushed and it seems gone now.